framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -587,6 +587,38 @@ public AnnotatedTypeFactory(BaseTypeChecker checker) {
             this.typeInformationPresenter = null;
         }
 
+        // Alias provided via -AaliasedTypeAnnos command-line option
+        // This can only be used for annotations whose attribute has the same name and order as the
+        // canonical.
+        // e.g. this will not be usable to declare an alias @Regex(string) for @Regex(value).
```

Comment:
```suggestion
        // This can only be used for annotations whose attributes have the same names as in the
        // canonical annotation, e.g. this will not be usable to declare an alias @Regex(index = 5) for @Regex(value = 5).
```

Reply:
Please run `./gradlew spotlessApply` to get line breaks right.